Zend/tests/pipe_operator/precedence_comparison.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Pipe binds higher than comparison
```

Comment:
The description does not appear to match the test itself (because the test contains parentheses).